src/memory.c

```diff
@@ -254,7 +260,7 @@ static void memory_scramble_rn(void)
 }
 
 
-uint8_t memory_setup(void)
+void memory_setup(void)
```

Comment:
Github doesn't allow commenting unchanged lines so I'll do it here. I see a couple calls to `memory_eeprom` which returns either `DBB_OK` or `DBB_ERROR`. Should those also be checked and call `HardFault_Handler` if the result is `DBB_ERROR`?

I didn't look up other functions. Maybe there are others.

Reply:
ACK
This is planned in a future commit that will be submitted after the current PRs are merged.